lib/CL/pocl_image_util.h

```diff
@@ -30,47 +30,60 @@
 #pragma GCC visibility push(hidden)
 #endif
 
-void origin_to_bytes (cl_mem mem, const size_t *origin, size_t *byte_offset);
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  void origin_to_bytes (cl_mem mem, const size_t *origin, size_t *byte_offset);
```

Comment:
This is an annoying flaw in the clang format. We should not indent due to the extern "C" nor namespace brackets. Please fix by hand unless you find a way to teach clang-format to not do this.